lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPlugin.java

```diff
@@ -89,10 +105,10 @@ public void start(InitializeParams params, XMLExtensionsRegistry registry) {
 		registry.registerDefinitionParticipant(definitionParticipant);
 	}
 
-	public void initialize() {
+	public void initialize(Object initializationOptions) {

```

Comment:
Angelo never confirmed if these initialization objects are always a JsonObject? Not a big deal though could be refactored in the future.